### PR TITLE
[Validator] added reviewed Latvian translations

### PR DESCRIPTION
**Pull request by k0d3r1s**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 1]
```diff
@@ -136,7 +136,7 @@
             </trans-unit>
             <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
-                <target state="needs-review-translation">Šī vērtība nav derīga IP adrese.</target>
+                <target>Šī nav derīga IP adrese.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf:139]: you're removing `vērtība` but we want the translation to be as if the english version where "This *value* is not a valid IP address." (same for other updates). Should we keep the word?

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf:139]: Or use `Šai vērtībai`  as in other existing messages that start with "This value is not"?

Comment (k0d3r1s) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf:139]: but source part now does not contain word "value". "vērtība" literally means "value"

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf:139]: yes, that's exactly what I mean.
See #53507

**Pull request closed without merging**